src/main/drivers/dshot.c

```diff
@@ -58,15 +58,15 @@
 void dshotInitEndpoints(const motorConfig_t *motorConfig, float outputLimit, float *outputLow, float *outputHigh, float *disarm, float *deadbandMotor3dHigh, float *deadbandMotor3dLow)
 {
     float outputLimitOffset = DSHOT_RANGE * (1 - outputLimit);
-    const float dshotIdleOffset = CONVERT_PARAMETER_TO_PERCENT(motorConfig->idleOffset * 0.01f);
+    const float dshotmotorIdle = CONVERT_PARAMETER_TO_PERCENT(motorConfig->motorIdle * 0.01f);
```

Comment:
```suggestion
    const float motorIdle = CONVERT_PARAMETER_TO_PERCENT(motorConfig->motorIdle * 0.01f);
```

Reply:
I'll use motorIdlePercent since it is clearer, as in:
`outputLow = DSHOT_MIN_THROTTLE + motorIdlePercent * DSHOT_RANGE;`
thanks for the suggestion.